platform-sdk/swirlds-common/src/main/java/com/swirlds/common/wiring/internal/SequentialWire.java

```diff
@@ -13,12 +29,16 @@
  */
 public class SequentialWire<T> implements Wire<T> {
     private final Consumer<T> consumer;
-    private final AtomicReference<Task> lastTask = new AtomicReference<>(new Task(true));
+    private final AtomicReference<Task<T>> lastTask;
```

Comment:
Consider renaming to `previousTask`, as "last" ambiguously can mean either "previous" or "final"